README.md

```diff
@@ -1 +1,78 @@
-# eslint-config-15gifts
\ No newline at end of file
+# eslint-config-15gifts
+
+This repository is a reusable eslint config for frontend projects at 15gifts. It includes as a base config (which can be used alone), and a React config that inherits the base config.
+It does make some opinionated tech choices (Typescript, EmotionCSS and Storybook for example) because it aims to make the config we have in projects smaller and more consistent.
+If you disagree with any of the rules or want to make a change, feel free to bring it up with the team and/or make a pull request.
+
+## Usage
+
+This project is not published on NPM but you can link straight to the github repo. Note that the version number is liste at the end. This correlates to the tag on a given commit (See https://github.com/15gifts/eslint-config-15gifts/tags).
```

Comment:
https://www.npmjs.com/package/@15gifts/eslint-config

Reply:
oh, but we don't actually use that anywhere do we? We should probably delete it?

Reply:
Yeah doesn't look like we actively use it since I think only a few people knew how to update it. I can't think of a reason why we'd need it.

Reply:
I've added a note about it, there are still a few projects pointing to it